group.yml

```diff
@@ -3,6 +3,7 @@ freeze_automation: scheduled
 vars:
   MAJOR: 4
   MINOR: 7
+  IMPACT: Low
```

Comment:
I didn't see Elliott PR https://github.com/openshift/elliott/pull/191/ actually references this variable. Better to remove this variable definition.